sdk/react/bottomOverlay.js

```diff
@@ -109,6 +123,37 @@ var BottomOverlay = React.createClass({
     return (<ProgressBar ref='progressBar' percent={percent} />);
   },
 
+  _getCuePointLeftOffset: function(cuePoint, progressBarWidth) {
+    var cuePointPercent = cuePoint / this.props.duration;
+    if (cuePointPercent > 1) {
+      cuePointPercent = 1;
+    }
+    if (cuePointPercent < 0) {
+      cuePointPercent = 0;
+    }
+    var leftOffset = this._renderLeftOffset(cuePointSize, cuePointPercent, progressBarWidth);
+    return leftOffset;
+  },
+
+  _renderCuePoints: function(cuePoints) {
+    var cuePointsView = [];
+    var progressBarWidth = this._renderProgressBarWidth();
+    var topOffset = this._renderTopOffset(cuePointSize);
+    var leftOffset = 0;
+    var positionStyle;
+    var cuePointView;
+
+    for (var cuePoint of cuePoints) {
```

Comment:
I would recommend sticking with the basic for syntax, as the for .. of throws an exception in iOS 7.